components/Modal/Modal.module.css

```diff
@@ -0,0 +1,25 @@
+.modal {
+    position: fixed;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+
+    background-color: var(--neutral-white);
+
+    z-index: 9 !important;
+
+    border-radius: 20px;
```

Comment:
Please use the global var css for it. If it doesn't exist, create one

Reply:
Created radius css variables

Reply:
Done